### DISCUSSION TITLE
Can I upgrade Talos Linux (1.7.0) with Existing Kubernetes Version (v1.28.3)?

### DISCUSSION THREAD

**Discussion by ghost** [siderolabs/talos#8634] [Q&A] [answered]
Currently I'm using Talos Linux `v1.6.1` with Kubernetes `v1.28.3` and I'm considering to Upgrade to Talos Linux `v1.7.0`

However, does Talos Linux `v1.7.0` supports Kubernetes `v1.28.3`?

Thanks.

Lee.

**Comment by steverfrancis** [comment, marked as answer]
Yes- you can see here
https://www.talos.dev/v1.7/introduction/support-matrix/
1.7 supports kubernetes 1.25 through 1.30

**Reply by ghost** [reply]
Oh my dear... I didn't search about coverages... Thank you very very much.
